### ISSUE TITLE
Positional parameter expansion issues

### ISSUE THREAD

**Issue by okdana** [jfcherng/Sublime-ShellScriptImproved#30]
Hi there. This is working really well for me, thanks for all the effort you've put into it.

One thing i've noticed though is that highlighting of parameter expansion 'operators' doesn't work well with positional parameters.

**Test case:**

```
: "${foo#*abc}"
: "${1#*abc}"
: "${foo##*abc}"
: "${1##*abc}"
: "${foo%%*abc}"
: "${1%%*abc}"
: "${foo:-abc}"
: "${1:-abc}"
: "${foo-abc}"
: "${1-abc}"
```

**Result:**

<img width="529" alt="screen shot 2016-10-09 at 00 08 40" src="https://cloud.githubusercontent.com/assets/122095/19218071/c7b624f6-8db4-11e6-969e-18fffcee432e.png">

Note that it treats suffix-`#` (beginning sub-string removal) as prefix-`#` (string length) when it encounters it in a positional parameter, which is why it accidentally looks semi-correct in the first pair of examples.

**Comment by jfcherng** [comment]
:+1: This package does not come up with positional parameter at all when you report this. It's created base on a test script provided by you iirc. 

![image](https://cloud.githubusercontent.com/assets/6594915/19219212/c22f2bc8-8e3f-11e6-98d7-b0ec79dcbb54.png)

Does the `*` before `abc` have a special meaning?

## 

Ref: [Bash Parameter expansion](http://wiki.bash-hackers.org/syntax/pe)

**Comment by okdana** [comment]
It's not part of the parameter expansion syntax itself,  but it is kind of special in the sense that any pattern following `#`, `##`, `%`, or `%%` is interpreted as a glob — so `foo='abcdef' echo "${foo#a*d}"` will print `ef`, for example. So you could highlight it as `keyword.operator.glob.shell`, if you wanted to be super thorough 💁

**Comment by jfcherng** [comment]
Should be resolved by b7c664dd310f59abaed94b8edec18fe6217b9827.

**Comment by okdana** [comment]
Thank you!
